assignments/array-methods.js

```diff
@@ -56,28 +56,71 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(array){
+    fullName.push(array.first_name + ' ' + array.last_name)
+})
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(function(drunkWithPower) {
+    allCaps.push(drunkWithPower.first_name.toUpperCase())
```

Comment:
```suggestion
    return drunkWithPower.first_name.toUpperCase()
```
What you have is working, but it is not exactly correct. The map() method creates a new array with the results of calling a provided function on every element in the calling array. The way you are using it is more like forEach. The forEach() method executes a provided function once for each array element, but does not return anything. So when we use forEach, and we want to create a new array from another array, we use push, like you did in the first problem, and then again here using map. However, since map returns a new array, we want to leverage that functionality. So we can just set allCaps equal to the new array being created by map. 
So if you look at the way I've re-written it here, we map over the runners array, and call each of it's elements drunkWithPower. Each of those elements runs through your provided callback, which returns that element uppercased. Map then returns an array of those elements, in the same order they were fed in. 
I hope this helps, let me know.